ui/js/controllers/logviewer.js

```diff
@@ -152,44 +157,34 @@ logViewerApp.controller('LogviewerCtrl', [
         };
 
         $scope.init = function() {
+
+            $scope.logProperties = [];
+            ThJobModel.get($scope.repoName, $scope.job_id).then(function(job) {
+                var jobStr = thJobSearchStr(job);
+                $scope.logViewerTitle = "Log for " + jobStr;
+                $scope.logProperties.push({label: "Job", value: jobStr});
+
+                $scope.logProperties.push({label: "Start", value: dateFilter(job.start_timestamp*1000, 'MMM d, y h:mm:ss a Z')});
+                $scope.logProperties.push({label: "End", value: dateFilter(job.end_timestamp*1000, 'MMM d, y h:mm:ss a Z')});
```

Comment:
I wonder if we should use `thDateFormat` similar to what we do [here](https://github.com/mozilla/treeherder/blob/29a704cc907ec8a64be02dc38d6bff5f555b5771/ui/plugins/controller.js#L223-L231) so the format is consistent with the rest of treeherder.